### PR TITLE
feat(org-stats): Wider intervals for <24h ranges and UI fixes

### PR DESCRIPTION
**Pull request by leedongwei**

### CODE REVIEW

File: static/app/views/usageStats/utils.tsx [hunk 1]
```diff
@@ -102,3 +109,34 @@ export function isDisplayUtc(datetime: DateTimeObject): boolean {
   const hours = parsePeriodToHours(interval);
   return hours >= 24;
 }
+
+/**
+ * Duplicate of getInterval from static/app/components/charts/utils
+ * except that we do not support <1h granularity for time ranges >6h
+ */
+export function getUsageInterval(datetimeObj: DateTimeObject) {
```
Comment (markstory) [inline static/app/views/usageStats/utils.tsx:117]: Is this something we could share with sessions as I think it has similar restrictions on granularity.

File: static/app/views/usageStats/utils.tsx [hunk 2]
```diff
@@ -102,3 +109,34 @@ export function isDisplayUtc(datetime: DateTimeObject): boolean {
...
+    // Greater than or equal to 30 days
+    return '4h';
+  }
+
+  if (diffInMinutes > ONE_HOUR * 6) {
+    // Between 6 hour and 30 days
+    return '1h';
+  }
+
+  if (diffInMinutes > ONE_HOUR) {
```
Comment (JoshFerge) [inline static/app/views/usageStats/utils.tsx:135]: we should probably not allow any intervals below an hour even for these small ranges, and in the future enforce that on the backend (unless someone asks for this feature) based on our slack discussion. is it possible to limit interval selection?

Comment (leedongwei) [reply static/app/views/usageStats/utils.tsx:135]: 😢

File: static/app/views/usageStats/utils.tsx (hunk 1 above)
Comment (leedongwei) [reply static/app/views/usageStats/utils.tsx]: Good call. Found the exact function in Sessions and I've moved them to `static/app/components/charts/utils.tsx`

**Review by JoshFerge — APPROVED** [review]

**Pull request merged**